examples/crm/src/deals/DealCreate.tsx

```diff
@@ -20,6 +20,7 @@ import { Deal } from '../types';
 export const DealCreate = ({ open }: { open: boolean }) => {
     const redirect = useRedirect();
     const dataProvider = useDataProvider();
+    const validateRequired = required();
```

Comment:
Why? If you want to extract, you should move it outside the component body as well imo :)

Reply:
You're right, that was the idea